splot/_viz_esda_mpl.py

```diff
@@ -1062,3 +1160,113 @@ def moran_loc_bv_scatterplot(moran_loc_bv, p=None,
         ax.plot(lag, fit.predy, **fitline_kwds)
         ax.scatter(moran_loc_bv.zy, fit.predy, **scatter_kwds)
     return fig, ax
+
+
+def moran_facette(moran_matrix, figsize=(16,12),
```

Comment:
I'd call it `moran_facet`, to align with other viz libraries ([seaborn](https://seaborn.pydata.org/api.html?highlight=facet#facet-grids) and [ggplot2](https://ggplot2.tidyverse.org/reference/facet_grid.html)).